qutebrowser/config/configtypes.py

```diff
@@ -806,6 +818,10 @@ def validate(self, value):
                 raise configexc.ValidationError(value, "may not be empty!")
         value = os.path.expanduser(value)
         try:
+            if not os.path.isabs(value):
+                abspath = os.path.join(standarddir.config(), value)
+                if os.path.isfile(abspath):
+                    return
```

Comment:
Is there a reason you set and check `abspath` separately instead of just doing `value = os.path.join(...)` and then using the existing check below?